turtleFSI/modules/domain.py

```diff
@@ -48,6 +55,19 @@ def assign_domain_properties(dx, dx_f_id, rho_f, mu_f, fluid_properties, dx_s_id
             solid_properties.append({"dx_s_id":dx_s_id,"material_model":material_model,"rho_s":rho_s,"mu_s":mu_s,"lambda_s":lambda_s})
     elif isinstance(solid_properties, dict): 
         solid_properties = [solid_properties]
+    
+    # RobinBC
+    if robin_bc==True:
+        ds_s = {}
+        if isinstance(ds_s_id, list): # If ds_s_id is a list (i.e, if there are multiple boundary regions):
+            for solid_boundaries in range(len(ds_s_id)):
+                ds_s[solid_boundaries] = ds(ds_s_id[solid_boundaries], subdomain_data=boundaries) # Create ds_s for each boundary
+                ds_s_ext_id_list=ds_s_id
```

Comment:
```suggestion
            for i, solid_boundaries in enumerate(ds_s_id):
                ds_s[i] = ds(solid_boundaries, subdomain_data=boundaries) # Create ds_s for each boundary
            ds_s_ext_id_list=ds_s_id
```
This looks cleaner and avoids reassigning `ds_s_id` multiple times